game/game.go

```diff
@@ -58,6 +58,7 @@ func (g *Game) moveAliens() {
 	for _, city := range g.cityMap {
 		cities := []string{}
 		if city.North != "" {
+			fmt.Println("going north")
 			cities = append(cities, city.North)
 		}
```

Comment:
**Suggestion:** Replace fmt.Println with a proper logging library for better control over log levels and formats.
```suggestion
 import "log"

 for _, city := range g.cityMap {
     cities := []string{}
     if city.North != "" {
         log.Println("going north")
         cities = append(cities, city.North)
     }
     if city.South != "" {
```